### PR TITLE
en-US Translation Corrections #1546

### PR DESCRIPTION
**Pull request by objecttothis**
This translation is complete. Hopefully no errors.

### CODE REVIEW

File: application/language/en-US/common_lang.php [hunk 1]
```diff
@@ -62,6 +62,6 @@
 $lang["common_view_recent_sales"] = "View Recent Sales";
 $lang["common_website"] = "website";
 $lang["common_welcome"] = "Welcome";
-$lang["common_welcome_message"] = "Welcome to OSPOS, click a module below to get started!";
-$lang["common_you_are_using_ospos"] = "You are using Open Source Point Of Sale Version";
-$lang["common_zip"] = "Zip";
+$lang["common_welcome_message"] = "Welcome to OSPOS, click a module below to get started.";
+$lang["common_you_are_using_ospos"] = "You are using Open Source Point Of Sale version";
+$lang["common_zip"] = "Postal Code";
```
Comment (daN4cat) [inline application/language/en-US/common_lang.php]: I was under the impression that US is ZIP not Postal Code. You sure this change is right?

File: application/language/en-US/customers_lang.php [hunk 2]
```diff
@@ -28,10 +28,10 @@
 $lang["customers_mailchimp_member_rating"] = "Rating";
 $lang["customers_mailchimp_status"] = "Status";
 $lang["customers_mailchimp_vip"] = "VIP";
-$lang["customers_max"] = "Max spent";
-$lang["customers_min"] = "Min spent";
+$lang["customers_max"] = "Max. spent";
```
Comment (daN4cat) [inline application/language/en-US/customers_lang.php]: not sure this is correct, I never seen Min and Max with a full stop. Uh?

File: application/language/en-US/common_lang.php (hunk 1 above)
Comment (objecttothis) [reply application/language/en-US/common_lang.php]: it's either "ZIP Code" which is specific to the United States Post Office or the more generic term "Postal Code" used also.  We can stick with "ZIP Code" but "Zip" is slang.  The reason I suggested Postal Code is non-native US English speakers would better understand it.  I don't feel strongly about ZIP Code vs Postal Code.

File: application/language/en-US/customers_lang.php (hunk 2 above)
Comment (objecttothis) [reply application/language/en-US/customers_lang.php]: US English abbreviation rules state that if you abbreviate a word it should be followed by a period.  International should be abbreviated Intl. Address should be abbreviated Addr. Street should be abbreviated St.

**Comment by SteveIreland** [comment]
Obviously an English teacher in a former life.  However min and max have become common words in their own right and no longer require a period.

File: application/language/en-US/common_lang.php (hunk 1 above)
Comment (daN4cat) [reply application/language/en-US/common_lang.php]: Not an issue for me, as long as @odiea and @SteveIreland are fine with it I'm fine with it.

File: application/language/en-US/customers_lang.php (hunk 2 above)
Comment (daN4cat) [reply application/language/en-US/customers_lang.php]: Yes, theoretically bu practically Min and Max are without dot. Anyway I left it like that, but on en-GB there is no dot.

**Pull request merged**